src/main/java/seedu/address/model/assessment/UniqueAssessmentList.java

```diff
@@ -55,6 +55,17 @@ public void add(Assessment toAdd) {
         internalList.add(toAdd);
     }
 
+    public Assessment getByName(AssessmentName name) {
```

Comment:
Should we specify the method as getbyName(AssessmentName name) throws AssessmentNotFoundException

Reply:
I think this one is okay, because AsssessmentNotFoundException is unchecked (only happens in runtime) so apparently we don't need to specify/handle hahaha